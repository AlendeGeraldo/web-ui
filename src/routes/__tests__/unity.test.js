import mainRoutes from '../main';

describe('Main Routes', () => {
  it('match Snapshot', () => {
    expect(mainRoutes).toMatchSnapshot();
  });
});
