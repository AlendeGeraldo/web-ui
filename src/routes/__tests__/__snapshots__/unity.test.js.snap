// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Routes match Snapshot 1`] = `
Array [
  Object {
    "component": Object {
      "$$typeof": Symbol(react.memo),
      "WrappedComponent": [Function],
      "compare": null,
      "displayName": "Connect(Root)",
      "type": [Function],
    },
    "exact": true,
    "icon": "home",
    "path": "/",
    "title": "In√≠cio",
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.memo),
      "WrappedComponent": [Function],
      "compare": null,
      "displayName": "Connect(Components)",
      "type": [Function],
    },
    "exact": true,
    "icon": "code",
    "path": "/components",
    "title": "Componentes",
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.memo),
      "WrappedComponent": [Function],
      "compare": null,
      "displayName": "Connect(ComponentDetail)",
      "type": [Function],
    },
    "notInMenu": true,
    "path": "/components/:componentId",
    "title": "Componente",
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.memo),
      "WrappedComponent": [Function],
      "compare": null,
      "displayName": "Connect(Projects)",
      "type": [Function],
    },
    "exact": true,
    "icon": "experiment",
    "path": "/projects",
    "title": "Projetos",
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.memo),
      "WrappedComponent": [Function],
      "compare": null,
      "displayName": "Connect(Project)",
      "type": [Function],
    },
    "notInMenu": true,
    "path": "/projects/:projectId/:experimentId?",
    "title": "Projeto",
  },
  Object {
    "component": [Function],
    "icon": "apartment",
    "path": "/implanted-flows",
    "title": "Fluxos Implantados",
  },
  Object {
    "component": [Function],
    "notInMenu": true,
    "path": "*",
  },
]
`;
